dressup/src/pages/ProductsPage/ProductsPage.data.js

```diff
@@ -2,11 +2,16 @@ import img1 from "../../images/jeans.jpeg";
 import img2 from "../../images/denim-jeans.jpeg";
 import img3 from "../../images/top.jpeg";
 import img4 from "../../images/jeans-two.jpeg";
+import img5 from "../../images/women-watch.jpeg";
+import img6 from "../../images/mens-watchOne.jpeg";
+import img7 from "../../images/MensWatch.webp";
+import img8 from "../../images/MensShirtTwo.webp";
+import img9 from "../../images/MensShirtThree.webp";
 
```

Comment:
Suggestion: you can import images in one file nd then imported from there, that will make file clean